src/translations/en.global.json

```diff
@@ -0,0 +1,12 @@
+{
+    "account.requestCode.success": "Sent SMS with code.",
+    "account.requestCode.error": "Failed to send SMS with code.",
+    "account.setupAccountRecovery.success": "Success",
+    "account.setupAccountRecovery.error": "Failed to setup account recovery.",
+    "account.recoverAccount.success": "Success",
+    "account.recoverAccount.error": "Failed to recover account.",
+    "account.create.success": "Congrats! this name is available.",
+    "account.create.error": "Username is taken. Try something else.",
+    "account.available.success": "User found.",
+    "account.available.error": "User not found."
+}
```

Comment:
just small suggestion: only "success" to setup and recover is not a good log, give every message with specific info to help logging

Reply:
that's not for logging, this is for end user

also these are default error messages, if error itself provides `messageCode` that should override this